src/styles/main.scss

```diff
@@ -1,3 +1,244 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+/* #region reset */
+
+* {
+  box-sizing: border-box;
+}
+
+h2,
+p {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
 body {
   margin: 0;
+
+  --accent-color: #00acdc;
+  --main-text-color: #060b35;
+}
+
+li {
+  list-style: none;
+
+}
+
+a {
+  text-decoration: none;
+  color: #000;
+}
+
+/* #endregion */
+
+.container {
+  max-width: 1200px;
+  padding: 0 50px;
+  margin: 0 auto;
+}
+
+.container--padding-s {
+  padding: 0 40px;
+}
+
+.header {
+  height: 60px;
+}
+
+.navigation__logo-link {
+  display: flex;
+  align-items: center;
+
+}
+
+.navigation {
+  position: relative;
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+}
+
+.navigation .navigation__logo-image {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: flex;
+}
+
+.nav__link {
+  padding: 22.5px 0;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  text-transform: uppercase;
+  position: relative;
+  display: flex;
+  align-items: center;
+  color: #060b35;
+}
+
+.nav__link:hover {
+  color: var(--accent-color);
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link.is-active {
+  color: var(--accent-color);
+}
+
+.nav__link.is-active:after {
+  content: "";
+  position: absolute;
+  height: 4px;
+  width: 100%;
+  display: inline-block;
+  background-color: var(--accent-color);
+  bottom: 0;
+  left: 0;
+  border-radius: 8px;
+}
```

Comment:
use nesting as you have this opportunity in scss
fix everywhere please
also create separated file for all BEM blocks